### PR TITLE
Lint markdown files to enforce consistent writing and reading experience

### PR DESCRIPTION
**Pull request by adamkudrna**
Let's make the “coding style” of markdown files consistent. Let's try the default settings first and see. There might be a later adjustment of line wrapping rules which have been subject to discussion recently.

### CODE REVIEW

File: .markdownlintrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "comment": "All rules, see https://github.com/DavidAnson/markdownlint#rules--aliases",
+
+  "default": true
```
Comment (mbohal) [inline .markdownlintrc:4]: I'm not sure about the default line length: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md013

IMHO 80 chars is really short. I would prefer to set it to 120 characters.

Comment (adamkudrna) [reply .markdownlintrc:4]: IMO: 80 is the best for reading, 120 makes reading difficult, and 100 doesn't hurt neither while editing, nor while reading. Let's go with 100, do you agree?

Comment (adamkudrna) [reply .markdownlintrc:4]: By the way, the recommended line length to achieve best possible readability is somewhere around 45–65 characters (sources may vary). 80 may be still acceptable for non-proportional type faces, 100 is on the edge. Compare `LICENSE.md` with `README.md` to see what reads easier. I'd like to get to 80 characters limit one day. But I believe it's good to get used to there is a limit first.

Comment (mbohal) [reply .markdownlintrc]: I dont really care…

**Pull request merged**